src/Icon/Icon.js

```diff
@@ -24,9 +27,14 @@ export default class Icon extends React.Component {
     };
   }
 
-  template({ icon, className, style, onClick }) {
-    return icon ?
-      <i className={cx('icon', `icon-${icon}`, className)} style={style} onClick={onClick}/>
+  template({ icon, className, style, onClick, paths }) {
+    return icon ? (
+      <i className={cx('icon', `icon-${icon}`, className)} style={style} onClick={onClick}>
+        {paths === 1 ? null : range(paths).map(k => (
```

Comment:
I grew up knowing that `React` doesn't like `false` as `children` but if you tell me that's ok I trust yoy